sagemaker/launch/launch_partition.py

```diff
@@ -1,15 +1,13 @@
 """ Launch SageMaker training task
 """
 import os
-import json
 import logging
 import boto3 # pylint: disable=import-error
 
 from sagemaker.pytorch.estimator import PyTorch
-from sagemaker.s3 import S3Downloader
 import sagemaker
 
-from common_parser import get_common_parser, parse_estimator_kwargs
+from common_parser import get_common_parser, parse_estimator_kwargs # pylint: disable=wrong-import-order
```

Comment:
I think this is a false positive of pylint, what I'm seeing is:

> third party import "from common_parser import get_common_parser, parse_estimator_kwargs" should be placed before "from sagemaker.pytorch.estimator import PyTorch"

So it thinks `from common_parser` is a third-party import, and `from sagemaker ...`  is a local import, possibly because the root directory is named `sagemaker`.  Because it wants third-party imports to be placed before local imports, we get this false positive here.